libtransmission/torrent.h

```diff
@@ -97,6 +97,12 @@ struct tr_torrent final : public tr_completion::torrent_view
         void on_verify_started() override;
         void on_piece_checked(tr_piece_index_t piece, bool has_piece) override;
         void on_verify_done(bool aborted) override;
+        void on_file_ok(tr_file_index_t) override
+        {
+        }
+        void on_file_error(tr_file_index_t, std::string) override
+        {
+        }
```

Comment:
clang-tidy will complain about these parameters having no name.

You can do something like `tr_file_index_t /*index*/` to avoid the warnings.